scripts/skills/injury/rf_dislocated_jaw_injury.nut

```diff
@@ -0,0 +1,81 @@
+this.rf_dislocated_jaw_injury <- this.inherit("scripts/skills/injury/injury", {
+	m = {
+		FatigueRecoveryModifier = -3,
+		ResolveMult = 0.7,
```

Comment:
Should we instead call this `BraveryMult`? At least that way in the code it is always called the same thing (Bravery instead of Resolve).

Reply:
I don't like that for Bravery/Resolve in particular and would leave it up to the respective dev.

By that logic we would need to start using `Stamina` for some fatigue related variable names. Though on a side note I actually want that example because I still want to partly return to `Stamina`

Reply:
In my opinion, when a multiplier applies to a certain property/field in an object it should conform to the naming scheme of THAT object. So, for example. if you want to modify the `_properties.Stamina` then you should use `StaminaMult` and if you want to modify the `actor.setFatigue` then you should use `FatigueModifier` etc.

This is the most stable and uniform way of working with these fields in my opinion and is the path of least resistance. Otherwise, everyone uses their own preferred style and it becomes a mess to browse through the codebase.

Reply:
I'm ok with this way of writing those typical member variables now